packages/office-ui-fabric-react/src/components/Panel/Panel.doc.tsx

```diff
@@ -14,7 +14,6 @@ import { PanelLightDismissExample } from './examples/Panel.LightDismiss.Example'
 import { PanelLightDismissCustomExample } from './examples/Panel.LightDismissCustom.Example';
 import { PanelMediumExample } from './examples/Panel.Medium.Example';
 import { PanelNonModalExample } from './examples/Panel.NonModal.Example';
-import { PanelPreventDefaultExample } from './examples/Panel.PreventDefault.Example';
```

Comment:
Why are we removing this example?

Reply:
I didn't see anything about this example on the a11y pass bug list

Reply:
There were 2 issues
1. in the a11y pass list, bug 4 was related to this panel in scan mode. 
2. This example didn't work at all

As the goal of this examples page is to highlight the panel, this example felt tangential - it's more about DOM event handling and less about how to interact with the panel.